nautobot/core/views/generic.py

```diff
@@ -255,17 +254,17 @@ def get(self, request):
                 self.queryset = self.queryset.none()
 
             display_filter_params = [
-                check_filter_for_display(filterset_filters, field_name, values)
+                check_filter_for_display(filterset.filters, field_name, values)
```

Comment:
Just checking - is `filterset.filters` equivalent to `filterset.get_filters()`, or are there subtle differences we need to be aware of?